### PR TITLE
Update issues-and-projects.yml

### PR DESCRIPTION
**Pull request by queenofcorgis**
@natalierjackson

### CODE REVIEW

**Comment by runtana** [comment]
Ok

**Review by natalierjackson — APPROVED** [review]

File: .github/DISCUSSION_TEMPLATE/issues-and-projects.yml [hunk 1]
```diff
@@ -26,10 +26,10 @@ body:
   - type: checkboxes
     attributes:
       label: Select Feature Area
-      description: What GitHub feature area would you like to discuss?
+      description: What GitHub product/feature area would you like to discuss? This category is only for conversations related to the below.
```
Comment (Manbiz1) [inline .github/DISCUSSION_TEMPLATE/issues-and-projects.yml:29]: G

Comment (Manbiz1) [inline .github/DISCUSSION_TEMPLATE/issues-and-projects.yml:29]: Gi

File: .github/DISCUSSION_TEMPLATE/issues-and-projects.yml [hunk 2]
```diff
@@ -26,10 +26,10 @@ body:
   - type: checkboxes
     attributes:
       label: Select Feature Area
```
Comment (Sandrakay49) [inline .github/DISCUSSION_TEMPLATE/issues-and-projects.yml:28]: You guys are the only help I've got I've been fighting for 2 years for my grandkids and I have known one but you guys   am getting drained stressed and worried them babe are my life.  If I can't get the evidence on them and I lose them  honestly I don't think I could live with out my grandkids and I read stuff last night that in my file that I can't have these CSS cps workers need to be stop save the children please

**Pull request merged**